### ISSUE TITLE
Docker images not being run

### ISSUE THREAD

**Issue by szeck87** [Glavin001/atom-beautify#1888]
# Description
I have Docker installed along with an image of unibeautify/autopep8.  When beautifying Python, I get an error stating autopep8 is not installed.  From the looks of it, it's trying to go through the cmd line instead of running the docker image.  There is an error before it says it could not find autopep8 that looks relevant.

```
10/12/2017 11:56:53 AM - ./Docker Executable [error]:  TypeError: Cannot read property 'slice' of null
    at Executable.parse [as versionParse] (executable.coffee [sm]:280)
    at executable.coffee [sm]:51
    at tryCatcher (/Users/stevenzeck/Desktop/Development/atom-beautify/node_modules/bluebird/js/release/util.js:16)
    at Promise._settlePromiseFromHandler (/Users/stevenzeck/Desktop/Development/atom-beautify/node_modules/bluebird/js/release/promise.js:512)
    at Promise._settlePromise (/Users/stevenzeck/Desktop/Development/atom-beautify/node_modules/bluebird/js/release/promise.js:569)
    at Promise._settlePromiseCtx (/Users/stevenzeck/Desktop/Development/atom-beautify/node_modules/bluebird/js/release/promise.js:606)
    at Async._drainQueue (/Users/stevenzeck/Desktop/Development/atom-beautify/node_modules/bluebird/js/release/async.js:138)
    at Async._drainQueues (/Users/stevenzeck/Desktop/Development/atom-beautify/node_modules/bluebird/js/release/async.js:143)
    at Async.drainQueues (/Users/stevenzeck/Desktop/Development/atom-beautify/node_modules/bluebird/js/release/async.js:17)
    at process._tickCallback (internal/process/next_tick.js:103)
```

# Input Before Beautification

This is what the code looked like before:

```
<INSERT CODE HERE>
```

# Expected Output

The beautified code should have looked like this:

# Actual Output

The beautified code actually looked like this:

```
<INSERT CODE HERE>
```

# Steps to Reproduce

1. Install Docker along with the unibeautify/autopep8 image, restart Atom
2. Add Python code to Atom editor
3. Run command `Atom Beautify: Beautify Editor`
4. Errors

# Debug

Here is a link to the `debug.md` Gist: <INSERT GIST HERE>

# Checklist

I have:
- [X] Tried uninstalling and reinstalling Atom Beautify to ensure it installed properly
- [X] Reloaded (or restarted) Atom to ensure it is not a caching issue
- [X] Searched through existing Atom Beautify Issues at https://github.com/Glavin001/atom-beautify/issues
so I know this is not a duplicate issue
- [ ] Filled out the Input, Expected, and Actual sections above or have edited/removed them in a way that fully describes the issue.
- [ ] Generated debugging information by executing `Atom Beautify: Help Debug Editor` command in Atom and added link for `debug.md` Gist to this issue

**Comment by szeck87** [comment]
Docker had a couple of recent releases, and `docker --version` outputs `Docker version 17.09.0-ce, build afdb6d4` right now.  https://github.com/Glavin001/atom-beautify/blob/master/src/beautifiers/executable.coffee#L382 doesn't look to recognize the `.0` ending to the version.

**Comment by szeck87** [comment]
Merged in #1889

**Comment by Glavin001** [comment]
Published to v0.30.6
